debug/DebugParser.js

```diff
@@ -0,0 +1,189 @@
+
+const util = require('util')
+const chalk = require('chalk')
+const StringBuffer = require('stringbuffer')
+
+const Tokenizer = require('../tokenization/Tokenizer')
+const AddressParser = require('../parser/AddressParser')
+
+class DebugParser {
```

Comment:
:+1:

Reply:
as a minor nit, the two lines above `AddressParser` and `DebugParser` seem to refer to the same concept, ie. a `*Parser`.

but it seems as though the two classes are in different directories and perform different functionality.

Reply:
Are you OK with the renaming @missinglink ? Good to merge ?